benchmark-ph1ex3/neutronics/convert.py

```diff
@@ -0,0 +1,2493 @@
+        XS.append(XSi)
+        SP0.append(SP0i)
+        kappa.append(kappai)
+    vi = 1/8 * np.ones(8)
+    homogenize_fuel(XS, SP0, kappa, vi, base)
+    print('Layer 8 done')
+
+    # F1 layer 9
+    base = directory + '/mhtgr_F1l9_'
+    XS = []
+    SP0 = []
+    kappa = []
+    for index in range(177, 181):
+        XSi, SP0i, kappai = getfuelxs('xsfiles/fuel171-180.xs', index)
+        XS.append(XSi)
+        SP0.append(SP0i)
+        kappa.append(kappai)
+    for index in range(181, 183):
+        XSi, SP0i, kappai = getfuelxs('xsfiles/fuel181-190.xs', index)
+        XS.append(XSi)
+        SP0.append(SP0i)
+        kappa.append(kappai)
+    vi = 1/6 * np.ones(6)
+    homogenize_fuel(XS, SP0, kappa, vi, base)
+
+    # F2 layer 9
+    base = directory + '/mhtgr_F2l9_'
+    XS = []
+    SP0 = []
+    kappa = []
+    for index in range(183, 191):
+        XSi, SP0i, kappai = getfuelxs('xsfiles/fuel181-190.xs', index)
+        XS.append(XSi)
+        SP0.append(SP0i)
+        kappa.append(kappai)
+    vi = 1/8 * np.ones(8)
+    homogenize_fuel(XS, SP0, kappa, vi, base)
+
+    # F3 layer 9
+    base = directory + '/mhtgr_F3l9_'
+    XS = []
+    SP0 = []
+    kappa = []
+    for index in range(191, 199):
+        XSi, SP0i, kappai = getfuelxs('xsfiles/fuel191-200.xs', index)
+        XS.append(XSi)
+        SP0.append(SP0i)
+        kappa.append(kappai)
+    vi = 1/8 * np.ones(8)
+    homogenize_fuel(XS, SP0, kappa, vi, base)
+    print('Layer 9 done')
+
+    # F1 layer 10
+    base = directory + '/mhtgr_F1l10_'
+    XS = []
+    SP0 = []
+    kappa = []
+    for index in range(199, 201):
+        XSi, SP0i, kappai = getfuelxs('xsfiles/fuel191-200.xs', index)
+        XS.append(XSi)
+        SP0.append(SP0i)
+        kappa.append(kappai)
+    for index in range(201, 205):
+        XSi, SP0i, kappai = getfuelxs('xsfiles/fuel201-210.xs', index)
+        XS.append(XSi)
+        SP0.append(SP0i)
+        kappa.append(kappai)
+    vi = 1/6 * np.ones(6)
+    homogenize_fuel(XS, SP0, kappa, vi, base)
+
+    # F2 layer 10
+    base = directory + '/mhtgr_F2l10_'
+    XS = []
+    SP0 = []
+    kappa = []
+    for index in range(205, 211):
+        XSi, SP0i, kappai = getfuelxs('xsfiles/fuel201-210.xs', index)
+        XS.append(XSi)
+        SP0.append(SP0i)
+        kappa.append(kappai)
+    for index in range(211, 213):
+        XSi, SP0i, kappai = getfuelxs('xsfiles/fuel211-220.xs', index)
+        XS.append(XSi)
+        SP0.append(SP0i)
+        kappa.append(kappai)
+    vi = 1/8 * np.ones(8)
+    homogenize_fuel(XS, SP0, kappa, vi, base)
+
+    # F3 layer 10
+    base = directory + '/mhtgr_F3l10_'
+    XS = []
+    SP0 = []
+    kappa = []
+    for index in range(213, 221):
+        XSi, SP0i, kappai = getfuelxs('xsfiles/fuel211-220.xs', index)
+        XS.append(XSi)
+        SP0.append(SP0i)
+        kappa.append(kappai)
+    vi = 1/8 * np.ones(8)
+    homogenize_fuel(XS, SP0, kappa, vi, base)
+    print('Layer 10 done')
+
```

Comment:
```suggestion
    print('Layer 10 done')
    return
```
I won't mark all of them... but you get the idea.